src/BigAssApi.js

```diff
@@ -0,0 +1,94 @@
+const dgram = require("dgram");
+const BigAssFan = require('./BigAssFan');
+const BigAssLight = require('./BigAssLight');
+const BigAssProperty = require('./BigAssProperty');
+const { syncingCallback, retryCall, myLogWrapper } = require('./utils');
+
+function FanMaster (numberOfExpectedDevices) {
+    this.allDevices = {}; // Dictionary of fan name -> BigAssDevice
+    this.sonnectionOpen = false;
```

Comment:
Typo - this is supposed to be `connectionOpen` - typo made in refactoring

Reply:
:heavy_check_mark: